src/pages/public_registration/RegistrationLandingPage.tsx

```diff
@@ -22,24 +24,48 @@ export const RegistrationLandingPage = () => {
   const { identifier } = useParams<IdentifierParams>();
   const user = useSelector((store: RootState) => store.user);
 
-  const registrationQuery = useQuery({
+  const registrationQuery = useQuery<Registration, AxiosError<DeletedRegistrationProblem>>({
     queryKey: ['registration', identifier],
     queryFn: () => fetchRegistration(identifier),
-    onError: () => dispatch(setNotification({ message: t('feedback.error.get_registration'), variant: 'error' })),
+    meta: {
+      handleError: (
+        error: AxiosError<DeletedRegistrationProblem>,
+        query: Query<Registration, AxiosError<DeletedRegistrationProblem>>
+      ) => {
+        if (error.response?.status === 410) {
+          const errorRegistration = query.state.error?.response?.data?.resource;
+          if (errorRegistration) {
+            query.setData(errorRegistration);
+          }
+        } else {
+          dispatch(
+            setNotification({
+              message: t('feedback.error.get_registration'),
+              variant: 'error',
+            })
+          );
+        }
+      },
+    },
   });
 
   const registration = registrationQuery.data;
   const registrationId = registration?.id ?? '';
 
   const isRegistrationAdmin =
     userIsRegistrationOwner(user, registration) || userIsRegistrationCurator(user, registration);
-  const isAllowedToSeePublicRegistration = registration?.status === RegistrationStatus.Published || isRegistrationAdmin;
+  const isAllowedToSeePublicRegistration =
+    registration?.status === RegistrationStatus.Published ||
+    isRegistrationAdmin ||
+    registration?.status === RegistrationStatus.DraftForDeletion ||
+    registration?.status ||
+    RegistrationStatus.Unpublished;
```

Comment:
`registration?.status` vil alltid sette denne til true så lenge den har en verdi (som den alltid vil ha)

```suggestion
    registration?.status === RegistrationStatus.Unpublished;
```